python_tests/pytest_input_generators.py

```diff
@@ -955,28 +955,28 @@ def permute_error_generator(
     # TODO Add duplicate axis check.
     yield SampleInput(
         make_arg(input_shape), [0, 1, 1, 3]
-    ), RuntimeError, "Duplicate entries in transformation map"
+    ), RuntimeError, "duplicated dimension entries"
 
     # TODO Add in-range axis check.
     yield SampleInput(
         make_arg(input_shape), [0, 1, 2, 4]
-    ), RuntimeError, "New2Old axes are not within the number of dimensions of the provided domain"
+    ), RuntimeError, "dims argument is out of range, expects"
```

Comment:
You can use `(.*?)` to match any string, which is useful for error handling checks.